### PR TITLE
Icon customization properties added to confirmation popup. Issue #11511

### LINKED ISSUE #11511: Add icon customization ability to confirmation popup
- [x] Default icon classes must be replaceable
- [x] Should be able to give custom icon with parameter

### PR DESCRIPTION
**Pull request by mahmut-gundogdu**
Icon customization properties added to confirmation service options. Now you can set custom icon or class to icon.

Resolves  #11511

### CODE REVIEW

File: npm/ng-packs/packages/theme-shared/src/lib/models/confirmation.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { LocalizationParam } from '@abp/ng.core';
+import {TemplateRef} from "@angular/core";
```
Comment (bnymncoskuner) [inline npm/ng-packs/packages/theme-shared/src/lib/models/confirmation.ts]:
```suggestion
```

**Review by bnymncoskuner — APPROVED** [review]

File: npm/ng-packs/packages/theme-shared/src/lib/models/confirmation.ts (hunk 1 above)
Comment (mahmut-gundogdu) [reply npm/ng-packs/packages/theme-shared/src/lib/models/confirmation.ts]: It was fixed.

File: docs/en/UI/Angular/Confirmation-Service.md [hunk 2]
```diff
@@ -75,7 +75,10 @@ const options: Partial<Confirmation.Options> = {
   yesText: 'Confirm',
   messageLocalizationParams: ['Demo'],
   titleLocalizationParams: [],
-};
+  // You can customize icon 
+  // icon: 'fa fa-exclamation-triangle', // or
+  // iconTemplate : '<img src="custom-image-path.jpg" alt=""/>'
+}
```
Comment (rqx110) [inline docs/en/UI/Angular/Confirmation-Service.md:81]:
```suggestion
};
```

File: npm/ng-packs/packages/theme-shared/src/lib/services/confirmation.service.ts [hunk 3]
```diff
@@ -17,13 +18,17 @@ export class ConfirmationService {
...
+    private contentProjectionService: ContentProjectionService,
+    @Inject(CONFIRMATION_ICONS) private confirmationIcons: ConfirmationIcons,
+  ) {}
 
   private setContainer() {
     this.containerComponentRef = this.contentProjectionService.projectContent(
       PROJECTION_STRATEGY.AppendComponentToBody(ConfirmationComponent, {
         confirmation$: this.confirmation$,
         clear: this.clear,
+        icons: this.confirmationIcons,
```
Comment (mahmut-gundogdu) [inline npm/ng-packs/packages/theme-shared/src/lib/services/confirmation.service.ts]: I will move to component

**Review by muhammedaltug — APPROVED** [review]

**Pull request merged**